### ISSUE TITLE
Support for edit/update?

### ISSUE THREAD

**Issue by downeymj** [immers-space/guppe#67]
Hi there. As I understand it (correct me if I'm wrong) the code now supports removal of the relayed post (toot) if the original author issues a delete.

Is the same support in place for the activitypub update activity (Mastodon edit feature)?

Also (although I suppose it's probably more delete-related) wonder about the newer auto-delete feature in Mastodon and if that would propagate out through the relayed posts as well.

**Comment by wmurphyrd** [comment]
Yes, Guppe is now based on my activitypub-express library, which implements the full ActivityPub spec, including Update.

However, Guppe is not involved in transmitting the contents of your posts to group members. It only relays the unique identifier for your post, and then other members' servers will request the content from your server directly, so the fact that Guppe applies the edit to its own copy of your post doesn't help the real issue of getting those updates to all the group members.

It really depends on how Mastodon addresses the update. If it sends the update to Guppe and addresses it to the group's followers, the Guppe will forward it along to all group members, as required in the Forwarding From Inbox section ActivityPub.
